### ISSUE TITLE
test ios devices 

### ISSUE THREAD

**Issue by ulisoaqa** [tmobile/stf_ios_support#51]
can you video test running de stf ios devices 
or pictures, i have incovenient in configure my enviroment :s

regards from mexico

**Comment by nanoscopic** [comment]
Yes; a demo both of setting everything up as well as showing it working is a good idea. It has not yet been created because there are still continuing changes to the setup process as things are altered and improved and the video / steps would quickly become out of date.

I will keep this in mind and update this ticket with links to video of it when created / available.

**Comment by KrzysztofKupczyk** [comment]
Hi, any update with video documentation?

**Comment by nanoscopic** [comment]
At this moment I am working on a complete replacement for STF for IOS devices.
It will be at MVP state within a few days.

https://github.com/nanoscopic/controlfloor is the new project that effectively replaces STF. It is written fully in Golang, and much more modern / simple in architecture design.

https://github.com/nanoscopic/ios_remote_provider is the new equivalent of this repo for use with controlfloor. It fixes many issues with stf_ios_support coordinator, simplifying how everything works and makes it all more stable/easy to start/use...

Zero documentation thus far. Essentially you run controlfloor, and point ios_remote_provider at it then start it also. Your ios devices then appear in controlfooor. Unlike STF, the devices will appear there whether video is working or not, and there will be fallback for video in case the primary method doesn't work...
